saiph/inverse_transform.py

```diff
@@ -36,16 +32,7 @@ def inverse_transform(
     """
     random_gen = np.random.default_rng(model.seed)
 
-    # Check dimension size regarding N
-    n_dimensions = len(model.dummy_categorical) + len(model.original_continuous)
-    n_records = len(coord)
-
-    if not use_approximate_inverse and n_records < n_dimensions:
-        raise InvalidParameterException(
-            f"n_dimensions ({n_dimensions}) is greater than n_records ({n_records})."
-        )
     # Get back scaled_values from coord with inverse matrix operation
```

Comment:
I don't understand what is the approximate inverse if this is the only place where we used it

Reply:
The inverse transform can't be in horizontal matrix as the coordinates are at worst squared. 
and we use https://numpy.org/doc/stable/reference/generated/numpy.linalg.pinv.html to do the inverse as https://numpy.org/doc/stable/reference/generated/numpy.linalg.inv.html
is only on squared matrix